src/views/HousingLottery/HousingLottery.module.scss

```diff
@@ -1,6 +1,6 @@
 .submit_preference_button {
-  position: "absolute";
-  margin: "auto";
```

Comment:
I'm pretty sure that you do not need the single quotes around absolute and auto.